src/push/kibana_api.ts

```diff
@@ -37,7 +37,8 @@ import {
 import { generateURL } from './utils';
 
 // Default chunk size for bulk put / delete
-export const CHUNK_SIZE = parseInt(process.env.CHUNK_SIZE) || 100;
+export const CHUNK_SIZE = parseInt(process.env.CHUNK_SIZE) || 250;
```

Comment:
We did a poor naming for this, lets change the variable name atleast that clarifies its monitors per push request